components/fieldgroups/experience-event/experienceevent-content.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-content",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Information",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Provides information content.",
```

Comment:
You can add more details in the description. For example, "Content related information for the content analytics"

Reply:
Followed some other examples, and now it's something like this: Content analytics data such as eventType, experience ID, and assets IDs.